Create CODEOWNERS

jayster312 opened the pull request:
# Description

Added the codeowners file to the root directory. Explains who is responsible for the code.

vanessa-cooper commented:
Reviewing this right now, let's see...

vanessa-cooper reviewed (changes requested):
Oh looks like the `CODEOWNERS` file I gave you is outdated. It has **Carl listed**, but he's... no longer with the company. Could you **remove him from the file**, but not from our hearts, and push that change?

jayster312 committed:
Create CODEOWNERS

Added the codeowners file to the root directory. Explains who is responsible for the code.

vanessa-cooper reviewed (approved):
Approved! Go ahead and merge this PR after all checks pass. Quickly now.

jayster312 committed:
Update CODEOWNERS

Removed Carl from the list.
